examples/gameserverallocation.yaml

```diff
@@ -46,11 +46,6 @@ spec:
         game: my-game
       matchExpressions:
       - {key: tier, operator: In, values: [cache]}
-      # [Stage:Beta]
-      # [FeatureFlag:StateAllocationFilter]
-      # Specifies which State is the filter to be used when attempting to retrieve a GameServer
```

Comment:
Same as above, please leave in the description.

Reply:
Included the description.